Signal/src/ViewControllers/ConversationView/ConversationInputToolbar.m

```diff
@@ -289,6 +292,12 @@ - (void)setQuotedReply:(nullable OWSQuotedReplyModel *)quotedReply
 
     // TODO animate
     [self.contentStackView insertArrangedSubview:self.quotedMessagePreview atIndex:0];
+    self.contentStackView.layoutMargins = UIEdgeInsetsMake(self.quotedMessageTopMargin, 0, 0, 0);
```

Comment:
So, usually we **don't** want the `contentStackView` to have a topMargin, but when it has a `quotedMessagePreview` we **do** want it to have a top margin, right?

It might be clearer to leave the contentStackView margins alone, and wrap the quotedMessagePreview in a view that includes the top margins. 

That way we don't have the inexplicable line 131 and we don't have to worry about cleaning it back up on line 311

Reply:
Sure, that's simpler. Done.